#! /bin/sh
patch -p0 -f $* < $0
exit $?

[sm]
2009-07-29  Marcus Brinkmann  <marcus@g10code.com>

	* keylist.c (print_capabilities): Print a trailing colon.

Index: keylist.c
===================================================================
--- sm/keylist.c	(revision 5080)
+++ sm/keylist.c	(working copy)
@@ -286,6 +286,8 @@
     es_putc ('S', fp);
   if ((use & KSBA_KEYUSAGE_KEY_CERT_SIGN))
     es_putc ('C', fp);
+
+  es_putc (':', fp);
 }
 
 
